sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/MetricsAdvisorClient.cs

```diff
@@ -356,8 +352,7 @@ Page<MetricSeriesDefinition> NextPageFunc(string nextLink, int? pageSizeHint)
         /// <exception cref="ArgumentException"><paramref name="metricId"/> is empty or not a valid GUID.</exception>
         public virtual AsyncPageable<MetricSeriesData> GetMetricSeriesDataAsync(string metricId, GetMetricSeriesDataOptions options, CancellationToken cancellationToken = default)
         {
-            Argument.AssertNotNullOrEmpty(metricId, nameof(metricId));
-            Argument.AssertNotNull(options, nameof(options)); // TODO: add validation for options.SeriesToFilter?
```

Comment:
`SeriesToFilter` (now called `SeriesKeys`) is a non-settable list (instantiated in the ctor as a `ChangeTrackingList`). Won't add an empty check, as mentioned before, and a null check is not necessary since it's always initialized.